src/util/errors-ts.ts

```diff
@@ -338,6 +338,19 @@ export class UnexpectedDomainPurchaseError extends NowError<
   }
 }
 
+/**
+ * Returned when there is an expected error charging the card.
+ */
+export class DomainPaymentError extends NowError<'DOMAIN_PAYMENT_ERROR', {}> {
+  constructor() {
+    super({
+      code: 'DOMAIN_PAYMENT_ERROR',
+      meta: {},
+      message: `Your card was declined.`
```

Comment:
Is the error always Card Declined?

Reply:
I don't have insight into our API so I can't tell. I assumed that a payment error always means charging the card fails which people usually refer to as the card being declined?